native/cocos/platform/interfaces/modules/canvas/CanvasRenderingContext2D.cpp

```diff
@@ -370,19 +370,33 @@ void CanvasRenderingContext2D::setTextBaseline(const ccstd::string &textBaseline
 
 void CanvasRenderingContext2D::setFillStyle(const ccstd::string &fillStyle) {
     CSSColorParser::Color color = CSSColorParser::parse(fillStyle);
+#if (CC_PLATFORM == CC_PLATFORM_MACOS || CC_PLATFORM == CC_PLATFORM_IOS || CC_PLATFORM == CC_PLATFORM_QNX || CC_PLATFORM == CC_PLATFORM_WINDOWS)
```

Comment:
- can only add macro for android, such as 
```c++
#if (CC_PLATFORM == CC_PLATFORM_ANDROID)
#else           
#endif  
```
- please add a comment why do special operation for android